edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -84,6 +84,11 @@ if /usr/bin/getent group docker >/dev/null; then
     %{_sbindir}/usermod -a -G docker %{iotedge_user}
 fi
 
+# Add iotedge user to systemd-journal group
+if /usr/bin/getent group systemd-journal >/dev/null; then
+    %{_sbindir}/usermod -a -G systemd-journal %{iotedge_user}
```

Comment:
`-a -G` -> `-aG`, and also add the "so it can get system logs" part to the comment.

Reply:
I was following the style of the line above, should I change it too? I will add the comment though

Reply:
I just went ahead and changed both